action.yml

```diff
@@ -5,8 +5,12 @@ branding:
   icon: cloud
   color: green
 inputs:
-  github_token:
-    description: 'Token for the repo. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
+  source_gh_token:
```

Comment:
can we keep the variable name  `github_token` here? Else it will be a breaking change. Ofc the env variable down in the file can be changed.

Reply:
Let me test it! Though I am still testing something at the moment with a private repo. There are still some problems with a private repo. Still figuring out why it is not working.

Reply:
I figured out why the private repo was not shown. That is fixed now but I think we have to stick with the different token names unfortunately. The name GitHub_Token is reserved 

![image](https://github.com/user-attachments/assets/4cb33ebe-ea36-4dc4-964a-d9184c8763e7)

Reply:
@AndreasAugustin the only way I could think of is to introduce a new variable for the target destination and unset the GITHUB_TOKEN when the source_token is explicitly. Otherwise I end all of the time with the error above. 

We would most likely need to update the test case I added to run on your private organization as well.

Reply:
@AndreasAugustin would the new approach of creating a new variable for the target next to the existing variable for the github_token work for you? What would be missing to get this merged and released :-)

Reply:
Hi @alexvanderberkel 
yes we can keep the name of the env var `TARGET_GTIHUB_TOKEN`. I was talking about the github action parameter :)

But nevertheless as mentioned before the explicit naming is way better. Lets stick to it and lets build a new main version